addons/redis/scripts/redis-sentinel-start-v2.sh

```diff
@@ -42,6 +42,76 @@ build_redis_sentinel_conf() {
   echo "build redis sentinel conf succeeded!"
 }
 
+check_register_sentinel_conf() {
+  if [ -f /data/sentinel/init_done.conf ]; then
+    echo "normal start"
+  else
+    echo "horizontal scaling"
+    touch /data/sentinel/init_done.conf
+    register_sentinel_conf
+  fi
+}
+
+register_sentinel_conf() {
+  if [[ -n "${SENTINEL_POD_NAME_LIST}" ]]; then
+    old_ifs="$IFS"
+    IFS=','
+    set -f
+    read -ra sentinel_pod <<< "${SENTINEL_POD_NAME_LIST}"
+    set +f
+    IFS="$old_ifs"
+
+    if [[ -n "${sentinel_pod[0]}" ]]; then
```

Comment:
you should consider all the sentinel nodes, not just sentinel_pod[0]

Reply:
I don't understand this part
Is it meant to be modified to `if [[ ${#sentinel_pod[@]} -gt 0 ]]; then`?